### DISCUSSION TITLE
cpg0003-rosetta

### DISCUSSION THREAD

**Discussion by ErinWeisbart** [broadinstitute/cellpainting-gallery#106] [Dataset descriptions]
Profile/data repo = data repurposed
Publication/analysis repo = https://github.com/carpenter-singh-lab/2022_Haghighi_NatureMethods

No images. Select numerical data in cellpainting-gallery workspace.

Project completion (dataset being made public):
~- [ ] Run [validation script](http://github.com/broadinstitute/cpg/cpgdata) to ensure data is completely uploaded~
- [x] Update cellpainting-gallery/README.md to add dataset to README

Once a paper is published:
- [ ] Update [Publications](https://github.com/broadinstitute/cellpainting-gallery/docs/publications.md)
- [ ] Add the paper to https://github.com/cytodata/awesome-cytodata

**Comment by ErinWeisbart** [comment]
The various datasets in the `workspace/preprocessed_data/DATASET/L1000` do not have the same selection of L1000 profiles. Is this intentional? It was pointed out to me in particular that not all datasets have treatment-level profiles and only LINCS has levels 3-5.

I quickly looked in the imaging-platform bucket to see if that offered any clarity and it did not. I looked at s3://imaging-platform/projects/2018_04_20_Rosetta/workspace/**preprocessed_data_gallery**/PROJECT/L1000/ and s3://imaging-platform/projects/2018_04_20_Rosetta/workspace/**preprocessed_data**/PROJECT/L1000/

Currently in CPG:
`CDRP-BBBC047-Bray`:
replicate_level_l1k.csv.gz
replicate_level_l1k_pclfc.csv.gz
replicate_level_l1k_pczscore.csv.gz
replicate_level_l1k_vczscore.csv.gz
treatment_level_l1k.csv.gz
(Matches preprocessed_data_gallery and preprocessed_data in imaging-platform.)

`CDRPBIO-BBBC036-Bray`
replicate_level_l1k.csv.gz
(There are additional profiles preprocessed_data, matches preprocessed_data_gallery in imaging-platform)

`LINCS-Pilot1`
level_3.csv.gz
level_4.csv.gz
level_4W.csv.gz
level_5_modz.csv.gz
level_5_rank.csv.gz
replicate_level_l1k.csv.gz
(Matches preprocessed_data_gallery and preprocessed_data in imaging-platform.)

`LUAD-BBBC041-Caicedo`
replicate_level_l1k.csv.gz
treatment_level_l1k.csv.gz
(Basically matches preprocessed_data_gallery and preprocessed_data in imaging-platform. In preprocessed_data there's an additional profile with _Juan suffix.)

`TA-ORF-BBBC037-Rohban`
replicate_level_l1k.csv.gz
replicate_level_l1k_QNORM.csv.gz
replicate_level_l1k_ZSPCQNORM.csv.gz
(Matches preprocessed_data_gallery and preprocessed_data in imaging-platform.)

**Reply by ErinWeisbart** [reply]
@shntnu Do you have any insight into this?

**Reply by shntnu** [reply]
To rephrase before I tag Marzieh on this:

For Cell Painting, we have the the same set of data levels available for all 5 datasets:

<img width="751" alt="Screenshot 2024-12-18 at 9 20 58 PM" src="https://github.com/user-attachments/assets/eaa9836f-7bd7-412a-82d2-a4d4b855c175" />

But that's not the case for L1000:

<img width="757" alt="Screenshot 2024-12-18 at 9 21 18 PM" src="https://github.com/user-attachments/assets/fba1098c-7441-492f-bdca-eaeb2f7c25c7" />

Is that the main concern @ErinWeisbart ?

**Reply by ErinWeisbart** [reply]
Yes, there are a different selection of L1000 profiles for each dataset.
1) It would be nice to have documented why they are different

I was also specifically asked if we could provide
2) treatment level for all datasets
3) other levels (3-5) for other datasets

**Reply by ErinWeisbart** [reply]
@MarziehHaghighi Can you help with the answer to this question?
A user would like to know why there are different levels of L1000 data available for the different projects? Were all the levels made for all the projects?

**Reply by shntnu** [reply]
@ErinWeisbart sorry I didn't follow up sooner. Indeed, Marzieh, your input would be key because I could not figure this out myself.
